### PR TITLE
CCS (docs) review of dashboard content

### PR DESCRIPTION
**Pull request by laubai**
Many minor and moderate changes to resolve:

- en-US language errors
- style problems (to improve clarity, readability)
- formatting errors (of things that customers need to copy and paste)
- incorrect steps (based on testing in current UI)
- incomplete or missing information

In particular, please review the TODO item in data/quickstarts/connect-rhosak-notebook-quickstart.yaml (adding a prerequisite) because I'm not sure if this is accurate, or if this tutorial might only be visible if Kafka is installed.

Please also review the Seldon tutorials, in particular any line with "Run a single prediction" because these were the sentences that were hardest to parse and I'm not 100% certain of their correctness.

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
@laubai Do you want to tag specific reviewers who might best be able to verify the textual changes?

**Comment by laubai** [comment]
@cfchase @sophwats You're probably the best folks to review these changes.

**Review by anishasthana — CHANGES_REQUESTED** [review]
@laubai can you please squash your commits?

**Comment by laubai** [comment]
@anishasthana Done, sorry about the wait.

**Comment by laubai** [comment]
I squashed the commits and there were conflicts so I rebased and now my commits are un-squashed and i'm having trouble re-squashing them. Any suggestions?

**Comment by jeff-phillips-18** [comment]
> I squashed the commits and there were conflicts so I rebased and now my commits are un-squashed and i'm having trouble re-squashing them. Any suggestions?

Looks good now.

**Review by anishasthana — COMMENTED** [review]
At a high level the changes look good to me, but I think we'll still want sign-off from someone like @cfchase  or @taneem-ibrahim

**Review by anishasthana — APPROVED** [review]
"approving" to get rid of the "requested changes" flag

File: data/quickstarts/deploy-python-model-quickstart.yaml [hunk 1]
```diff
@@ -8,69 +8,75 @@ spec:
...
-
-    We just know you will absolutely love this amazing environment.  This quick start will get your model deployed for sandbox development usage.
+    This quick start shows you how to take your model out of a Jupyter notebook and put it into a Flask application running on OpenShift Dedicated, for use as a development sandbox.
   tasks:
-    - title: Create a new repository from the s2i Flask github template
+    - title: Create a new GitHub repository using the s2i Flask template
       description: |-
-        ### To create a new repository from the 2si Flask template:
+        ### To create a new repository from the s2i Flask template:
         1. Navigate to `https://github.com/cfchase/s2i-flask-notebook`.
```
Comment (cfchase) [inline data/quickstarts/deploy-python-model-quickstart.yaml:19]: This repo has been transferred to https://github.com/opendatahub-io/odh-s2i-project-simple

**Review by cfchase — CHANGES_REQUESTED** [review]
One URL change, otherwise it looks great.  The Marketplace comment is food for thought, but not necessary for merge.

File: data/getting-started/seldon-deploy.md [hunk 2]
```diff
@@ -14,17 +14,17 @@ Features include:
...
+Seldon Deploy adds a user interface for easily deploying, monitoring and managing models. It adds enterprise functionality such as user roles and permissions, overall resource usage and integrations such as gitops.
 
 ## Installing Seldon Deploy
 
 Seldon Deploy has a Red Hat marketplace listing. That install includes Red Hat versions of required sub-components such as Seldon Core.
 
 Documentation on the Seldon Deploy Red Hat marketplace install is at [https://deploy-master.seldon.io/docs/getting-started/redhat-installation/](https://deploy-master.seldon.io/docs/getting-started/redhat-installation/)
 
-### Subscribe to the operator on Market place 
+### Subscribe to the operator on Market place
```
[the remark this answers is not in the document]
Comment (laubai) [reply data/getting-started/seldon-deploy.md:27]: Agreed, will update today.

File: data/quickstarts/deploy-python-model-quickstart.yaml (hunk 1 above)
Comment (laubai) [reply data/quickstarts/deploy-python-model-quickstart.yaml:19]: Thanks! Will update today.

**Review by cfchase — APPROVED** [review]

**Pull request merged**